pages/_en-US/ds-index/3ds-forwarders.md

```diff
@@ -13,12 +13,39 @@ tabs:
 If you have any issues, check the FAQs on the [GBAtemp thread](https://gbatemp.net/threads/nds-forwarder-cias-for-your-home-menu.426174/).
 {:.alert .alert-warning}
 
+HOME menu forwarders are icons on your HOME menu that redirect to another software that runs the icon as listed. In this case, one can add DS games from the SD card (utilizing nds-bootstrap) or from a compatible flashcard (via its respective kernel) to the HOME menu for easier access to games.
+
+DS games need to be dumped to a digital `.nds` format. You can dump your DS cartridges using [GodMode9](https://3ds.hacks.guide/godmode9-usage#dumping-a-game-cartridge).
+{:.alert .alert-info}
+
+This page assumes you are running a modern CFW environment from [3ds.hacks.guide](https://3ds.hacks.guide).
+
+Choose one of the following to add to the HOME menu:
+
+{% capture tab-sd-card %}
+
+### Part 1: Getting the required files
+1. Open FBI and select `Remote Install`, then `Scan QR Code`
+1. Scan this QR code to install the latest version of [Universal-Updater](https://github.com/Universal-Team/Universal-Updater)<br>
+   ![Universal-Updater QR code](https://db.universal-team.net/assets/images/qr/universal-updater.cia.png)
```

Comment:
Might be good to mention that you can skip this if you already have Universal-Updater as a lot of people probably already have it considering it's on 3ds.hacks.guide now and the default way to install TWiLight.